src/general/index.js

```diff
@@ -57,14 +57,24 @@ const influx = new Influx.InfluxDB({
                 connections: Influx.FieldType.INTEGER,
             },
             tags: ["host"]
-        }
+        },
+	{
+	measurement: 'gettxoutsetinfo;,
```

Comment:
Seems like you've missed a `'` here. 

```diff
-	measurement: 'gettxoutsetinfo;,
+ 	measurement: 'gettxoutsetinfo',
```

Reply:
That's what I get for re-typing it. Thanks